games/uMed.md

```diff
@@ -0,0 +1,7 @@
+---
+slug: https://koboldgames.ch/index.php/project/umed?lang=eng
+title: 'uMed: Christen Markus & Eichinger Tobias'
```

Comment:
```suggestion
title: 'uMed'
```

the game is called just uMed, seems a bit weird to put their names in the title :D and they are clearly labelled on the website we refer to

Reply:
:D I see - I thought that this is uncommon